tests/TestCase/Database/Query/UpdateQueryTest.php

```diff
@@ -471,13 +471,9 @@ public function testJsonValue(): void
         $driver = $this->connection->getDriver();
         $version = $this->connection->getDriver()->version();
         if ($driver instanceof Postgres && version_compare($version, '12.0.0', '<')) {
-            $skip == true;
```

Comment:
Aren't those meant to be assignment instead?

Reply:
Yes, it's used in the `skipIf()` call below.